controller/src/main/java/org/jboss/as/controller/AbstractWriteAttributeHandler.java

```diff
@@ -178,31 +176,6 @@ protected abstract void revertUpdateToRuntime(OperationContext context, ModelNod
             throws OperationFailedException;
 
     /**
-     * Does nothing. Subclasses can alter this behavior.
-     *
-     * @param attributeName the name of the attribute being updated
-     * @param unresolvedValue the unresolved value
-     *
-     * @deprecated the {@link AttributeDefinition} provided to the constructor will validate
-     */
-    @Deprecated
-    protected void validateUnresolvedValue(final String attributeName, final ModelNode unresolvedValue) throws OperationFailedException {
```

Comment:
removing this as they ware deprecated back in march 2013 and not used anywhere anymore.